pages/posts/2017/12-december/140-continuous-integration-at-subvisual.md

```diff
@@ -0,0 +1,116 @@
+---
+id: 140
+path: /posts/140-continuous-integration-at-subvisual/
+title: "Continuous Integration at Subvisual"
+author: miguel-palhas
+date: 14/12/2017
+tags:
+  - development
+intro: "Learn how we do Continuous Integration, and what tools we use at each step of the way"
+---
+
+We're often approached by other companies or developers who want to know more about how we handle our projects' testing and deployment.
+
+Especially since we build and ship multiple times a day and often have multiple developers working in multiple branches simultaneously.
+
+This post will give you the highlights of how all of is done at Subvisual.
+
+We don't claim to have that one methodology to rule them all, but hopefully you can extract something from our process and apply it to your own.
+
+So, off we go...
+
+# Automated builds
```

Comment:
What do you think moving this small section to the CI section on the bottom? You're basically just saying we do automated builds/deploys on Semaphore but you'll explain it later.